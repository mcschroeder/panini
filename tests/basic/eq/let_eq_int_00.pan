eq : (a:ℤ) → (b:ℤ) → { c:𝔹 | c = true ⟺ a = b }

f00 = let x = 1 in eq x 1

f01 : {v:bool|?}
f01 = let x = 1 in eq x 1

f02 : {v:bool|v=true}
f02 = let x = 1 in eq x 1


f10 = let x = 1 in eq 1 x

f11 : {v:bool|?}
f11 = let x = 1 in eq 1 x

f12 : {v:bool|v=true}
f12 = let x = 1 in eq 1 x
