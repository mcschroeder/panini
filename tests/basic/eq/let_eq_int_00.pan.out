f00 : {c:𝔹 | c = true}
f01 : {v:𝔹 | c = true}
f02 : {v:𝔹 | v = true}
f10 : {c:𝔹 | c = true}
f11 : {v:𝔹 | c = true}
f12 : {v:𝔹 | v = true}
